data/campaigns/The_South_Guard/utils/sg_deaths.cfg

```diff
@@ -49,21 +49,19 @@
     [/if]
 [/event]
 
-#ifndef SG_DEATHS_LAST_LEVEL
```

Comment:
This exists because in S9a Ethiliel has a custom death event.

In this campaign the death events are not included as macro, but instead this whole file is included into (nearly) each scenario. This allows different tricks, like hiding it here with an #ifdef.
It's an alternative way to code it.